Description: Do not crash if fbsplash failed to load and usplash is not enabled
Bug: https://github.com/NigelCunningham/Tuxonice-Userui/issues/5
Author: Andrey Rahmatullin <wrar@wrar.name>
Last-Update: 2012-04-28

--- a/userui_core.c
+++ b/userui_core.c
@@ -1043,16 +1043,17 @@
 	/* Initialise all that we can, use the first */
 //  active_ops = NULL;
 	for (i = 0; i < NUM_UIS; i++) {
-		if (userui_ops[i] && userui_ops[i]->load)
+		if (userui_ops[i] && userui_ops[i]->load) {
 			result = userui_ops[i]->load();
-		if (result) {
-			if (test_run)
-				fprintf(stderr, "Failed to initialise %s module.\n", userui_ops[i]->name);
-			else
-				printk("Failed to initialise %s module.\n", userui_ops[i]->name);
-		} else
-      if (!active_ops)
-        active_ops = userui_ops[i];
+			if (result) {
+				if (test_run)
+					fprintf(stderr, "Failed to initialise %s module.\n", userui_ops[i]->name);
+				else
+					printk("Failed to initialise %s module.\n", userui_ops[i]->name);
+			} else
+				if (!active_ops)
+					active_ops = userui_ops[i];
+		}
 	}
 
 	if (active_ops->prepare)
